tracks/pep8/lab1-1/tests.txt

```diff
@@ -0,0 +1,16 @@
+===
```

Comment:
I do not plan to keep this format. It just allow me to reuse existing Oto test files.